packages/ui/src/providers/Theme/index.tsx

```diff
@@ -55,20 +57,26 @@ export const defaultTheme = 'light'
 
 export const ThemeProvider: React.FC<{
   children?: React.ReactNode
-  cookiePrefix?: string
   theme?: Theme
-}> = ({ children, cookiePrefix, theme: initialTheme }) => {
-  const cookieKey = `${cookiePrefix || 'payload'}-theme`
+}> = ({ children, theme: initialTheme }) => {
+  const { config } = useConfig()
+
+  const preselectedTheme = config.admin.theme
+  const cookieKey = `${config.cookiePrefix || 'payload'}-theme`
```

Comment:
Explanatory comment:
cookiePrefix is ​​not passed as a prop, because config is needed to access `theme` anyway, and cookiePrefix is also ​​a config property.